Cross Team/Minutes/20221004.md

```diff
@@ -0,0 +1,43 @@
+# 6th cross-team meeting, 4 October 2022, 3-4pm CEST
+
+## Action log
+
+| ID | Owner | Action |
+|-|-|-|
+| 1.6 | Alistair | Wrapping up the discussion on backward compatibility |
+| 1.7 | Alistair, Birgit, Ranjini, Lisa | Recurring action: being on the lookout for user engagement funding opportunities |
+| 5.1 | Alistair | Progress preparation for implementation of CLA assistant | 
+| 5.2 | Alistair | Recurring action until CA is signed: keep progressing discussions with partners on CA | 
+| 5.3 | Birgit | Create first draft of work plan using workshop notes (GitHub discussions) |
+
+
+## Agenda
+1. Action items from last meeting
+2. Workshop programme & preparations
+3. Next steps for backward compatibility strategy
+4. Date of next meeting
+
+
+## Actions from last meeting
+| ID | Owner | Action | Report & new status |
+|-|-|-|-|
+| 1.6 | Alistair | Wrapping up the discussion on backward compatibility as described below | See agenda item below. Keep action open until the policy is finalised. |
+| 1.7 | Alistair, Birgit, Ranjini, Lisa | Being on the lookout for user engagement funding opportunities | Recurring action. No new items raised. |
+| 5.1 | Alistair | Progress preparation for implementation of CLA assistant | Test implementation demonstrated by Theo Geddes at Met Office. One niggle is the difficulty of sharing ownership of the gist on which appears to be tied to an individual. Suggestion to see if gist can be owned by an organisation, though would that open personal data to too many people? Action continued. |

```

Comment:
It looks like organizational gists are not supported by GitHub: https://github.com/community/community/discussions/7923